.gitignore

```diff
@@ -32,6 +32,7 @@ yarn-error.log*
 
 # env files (can opt-in for committing if needed)
 .env*
+!.env.local
```

Comment:
better to have smth like `.env.local.example` file with description of all environment variables, but .env file itself should not be exposed to repository